k8s/components/gridsuite/patches/cleanup-voltageinit-sidecar.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: not-important
+spec:
+  template:
+    spec:
+      containers:
+        - name: cleanup
+          image: busybox:1.36.1
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp-emptydir
+          imagePullPolicy: IfNotPresent
+          securityContext:
```

Comment:
runAsUser: 1000 may be necessary for it to have write access to the files written by the main container